README.md

```diff
@@ -47,7 +49,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+belle = town[:residents].delete(1)
+town.castle[:guests].push(belle)
 ```
```

Comment:
`.delete` expects an argument that matches a value in the array, not an index. In this case, it should be `belle = town[:residents].delete("Belle")`. If you wanted to remove something by index, you should use `.delete_at`.